.github/workflows/build-and-test-workflow.yml

```diff
@@ -0,0 +1,47 @@
+name: MDH UI Build and Test Workflow
+
+on:
```

Comment:
need to add `paths-ignore` to filter out the directory for the amplify app (`"src/**"`) so we don't run this on changes to the site.